### DISCUSSION TITLE
🔗 Edit on Mobile devices (Duplicate of #23528)

### DISCUSSION THREAD

**Discussion by PivitParkour94** [elementor/elementor#17612] [Feature Request]
Basically using Elementor on a mobile device is quite a nice experience. 

The problem comes when you want to do something that is limited to the right click menu. 

My proposal is everything that’s in the right click menu be added as a section in the advanced side bar as well. The other option is having the right click menu reprogrammed to work for mobile devices (like it works in figma for example). 

From a technical point of view this couldn’t be too difficult because the functionality is already there you just have to trigger that code in another spot. 

If this isn’t the right place to have a feature request of a technical nature for the Elementor page builder in WordPress,  if you could please point me in the right direction to get this in front of the developers responsible that’d be awesome.

Cheers,
Nathaniel

**Comment by nicholaszein** [comment]
Hello there, @PivitParkour94!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23528 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
